### PR TITLE
[W5.7][W14-3]Darien Chong Yung Han

### PR DESCRIPTION
**Pull request by darienchong**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -36,14 +52,41 @@
...
-    	sortKey = key;
+    	keyExtractorName = getKeyExtractorName(key);
+    }
+    
+    public String getKeyExtractorName(String key) {
+    	try {
+    		return SORTKEY_METHOD_NAMES[Arrays.asList(KEYWORDS).indexOf(key)];
+    	} catch (ArrayIndexOutOfBoundsException e) {
+    		return "";
+    	}
```
Comment (yuntongzhang) [inline src/seedu/addressbook/commands/SortCommand.java:72]: The indentation of this file is not consistent. Try fix the indentation here.

File: src/seedu/addressbook/commands/IncorrectCommand.java [hunk 2]
```diff
@@ -5,10 +5,18 @@
  * Represents an incorrect command. Upon execution, produces some feedback to the user.
  */
 public class IncorrectCommand extends Command {
+	
+	/** The feedback message to be shown to the user. */
```
Comment (lingzhiyu) [inline src/seedu/addressbook/commands/IncorrectCommand.java]: Fix indentation on comment

**Review by lingzhiyu — COMMENTED** [review]
Slight indentation mistakes.

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        CommandResult result = command.execute();
+
+        assertEquals(Command.getMessageForPersonListShownSummary(expectedPersonList), result.getFeedbackToUser());
+    }
+	
+	private void assertSortCommandInvalidKey(String keyword) {
+		SortCommand command = createSortCommand(keyword);
+		CommandResult result = command.execute();
+		
+		assertEquals(SortCommand.ILLEGAL_SORTKEY_MESSAGE, result.getFeedbackToUser());
```
Comment (kexiaowen) [inline test/java/seedu/addressbook/commands/SortCommandTest.java:60]: There are some compilation errors here. I think it should be `result.feedbackToUser` instead.

File: src/seedu/addressbook/commands/SortCommand.java (hunk 1 above)
Comment (kexiaowen) [reply src/seedu/addressbook/commands/SortCommand.java:72]: As pointed out by Yuntong, there are a lot of indentation issues in your files. Remember that basic indentation should be **4 spaces** and indentation for wrapped lines should be **8 spaces**

**Review by kexiaowen — APPROVED** [review]
@snajef Nice attempt! It is good that you also updated user guide, I/O and JUnit tests. You might also want to update HelpCommand.java as well. 
The major issue in this PR is coding style violation (indentation is wrong in all the files). This reduces readability of your code. You can close this PR after reading my comments.

**Pull request closed without merging**